inst/rmarkdown/templates/fairness_report/skeleton/skeleton.Rmd

```diff
@@ -31,6 +33,54 @@ In this fairness report, we investigate the fairness of the following task:
 task
 ```
 
+### The General Task Documentation for the task:
+
+```{r}
+# Create the report attributes
+report_attr = list(
+  "Audit Date: ",
+  "Task Name: ",
+  "Number of observations: ",
+  "Number of features: ",
+  "Target Name: ",
+  "Feature Names: ",
+  "The Protected Attribute: "
+)
+
+# Create the skeleton of summary table
+summary_table <- data.frame(Value=rep(NA, length(report_attr)))
+rownames(summary_table) <- report_attr
+
+#Assign the value to the table
+summary_table$Value = c(
+  as.character(Sys.Date()),
+  task$id,
+  task$nrow,
+  task$ncol,
+  task$target_names,
+  paste(task$feature_names, collapse = ", "),
+  task$col_roles$pta
+)
+
+summary_table %>%
+  kbl() %>%
+  kable_paper("hover", full_width = F)
+```
+### Audit Results: Summary
+TBD
+
+### Exploratory Data Analysis: 
```

Comment:
Have a look if anything from https://cran.r-project.org/web/packages/DataExplorer/vignettes/dataexplorer-intro.html can be helpful perhaps?